packages/appstore/src/components/withdraw-request-verification/withdraw-request-verification.tsx

```diff
@@ -0,0 +1,13 @@
+import React from 'react';
+import WithdrawalVerificationEmail from '@deriv/cashier/src/pages/withdrawal/withdrawal-verification-email';
+
+const WithdrawRequestVerification = () => {
+    return (
+        <WithdrawalVerificationEmail
+            request_email_icon='IcWithdrawRequestVerification'
+            sent_email_icon='IcWithdrawRequestVerificationSent'
```

Comment:
We don't need different icons for wallets and cashier. The plan is to update cashier with the icons from wallet design later when wallet changes are released.